gulp/build-tasks/modules.js

```diff
@@ -73,7 +73,7 @@ module.exports = function (gulp, plugins, config, gulpWebpack, webpack, path) {
           ],
         }, webpack))
         .pipe(gulp.dest(`${config.basepath.build}/assets/${config.versionName}/latest/modules/${element}`))
-        .pipe(plugins.replace(new RegExp('="/assets/', 'g'), '="assets/'))
+        .pipe(plugins.replace(new RegExp('\'/assets/', 'g'), '\'assets/'))
```

Comment:
Should this cover both types of quote marks?

Reply:
Yes @ThrawnCA , it is a regular expression which checks for **="/assets/..../"**